docs/src/pages/[platform]/connected-components/liveness/faqs-android.mdx

```diff
@@ -0,0 +1,7 @@
+### (Android-specific) When creating a backend using a REST API, seeing compilation errors relating to `com.amazonaws.mobileconnectors.apigateway.annotation`
+
+These autogenerated files can be safely deleted — this should be a one-time fix after `amplify add api` and `amplify push`. To read more, check out: [Configure Amplify API Android documentation](https://docs.amplify.aws/lib/restapi/getting-started/q/platform/android/#configure-api)
+
+### (Mobile-specific) I am unable to use Cognito. Can I still use the FaceLivenessDetector?
```

Comment:
Do we need to say "Mobile-specific" if it's already on the Android page?

```suggestion
### (Mobile-specific) I am unable to use Cognito. Can I still use the FaceLivenessDetector?
```

Reply:
I'm just imagining a customer trying to implement liveness on multiple platforms, starting with Swift/Android and then expecting to be able to follow the same steps and use the credentials provider on JS too.  maybe an extremely small use case but that's what's on my mind 😅

Reply:
I guess the conveyed information is that seeing this on a mobile page also tells the customer that this feature isn't accessible on web?